website/docs/r/container_cluster.html.markdown

```diff
@@ -184,6 +186,18 @@ maintenance_policy {
 }
 ```
 
+The `ip_allocation_policy` block supports:
+
+* `cluster_secondary_range_name` - (Optional) The name of the secondary range to be
```

Comment:
The last sentence of this (and the next) are no longer applicable.

Reply:
Done.